.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .pytest_cache/
 **__pycache__/
 db.sqlite3
+taxi/migrations
```

Comment:
The `taxi/migrations` entry should have a trailing slash to indicate that it is a directory. Consider changing it to `taxi/migrations/` to ensure the entire directory is ignored.